### ISSUE TITLE
Reword complex sentence

### ISSUE THREAD

**Issue by saunderd-netapp** [NetAppDocs/storagegrid-117#18]
### Page URL

https://docs.netapp.com/us-en/storagegrid-117/installconfig/optional-enabling-node-encryption.html

### Page title

Optional: Enable node encryption

### Summary

The sentence "Data that exists on an appliance that has node encryption enabled before it is connected to the KMS is encrypted with a temporary key that is not secure." is complex and hard to parse. I recommend rewording it to "Before an appliance with node encryption enabled is connected to the Key Management System (KMS), the appliance's data is encrypted with a temporary key that is not secure."

### Public issues must not contain sensitive information

- [X] This issue contains no sensitive information.

**Comment by netapp-pcarriga** [comment]
Hi @saunderd-netapp, thanks for providing feedback. I'll update the documentation accordingly.

**Comment by netapp-pcarriga** [comment]
Hi @saunderd-netapp, we made some further edits to your suggested change and it's now live on the StorageGRID 11.7 site. Thank you again for bringing this to our attention.
